pkgs/top-level/all-packages.nix

```
@@ -31010,6 +31010,8 @@ with pkgs;
 
   hushboard = python3.pkgs.callPackage ../applications/audio/hushboard { };
 
+  husky = callPackage ../tools/misc/husky { };
```

```suggestion
  husky = callPackage ../development/tools/misc/husky { };
```

This would be a better place.